styles/style.css

```diff
@@ -0,0 +1,421 @@
+  background-repeat: repeat;
+  background-size: contain;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+#intro-text {
+  padding-top: 20px;
+  padding-bottom: 20px;
+  margin-bottom: 20px;
+  padding-left: 10px;
+  background-color: #f2f2f292;
+  border: 4px solid white;
+}
+
+#date-location {
+  padding-top: 20px;
+  font-size: 18px;
+}
+
+/* STATS */
+#stats {
+  background-image: url('../images/stats-bg.jpeg');
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  position: relative;
+}
+
+#stats-grid {
+  display: grid;
+  gap: 10px;
+}
+
+#stats-heading {
+  color: #ec5242;
+  margin-bottom: 30px;
+  border-bottom: 3px solid white;
+}
+
+.stats-card {
+  background-color: #5f5f5f50;
+  display: flex;
+  padding: 20px 10px;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  gap: 20px;
+  transition: all 0.75s ease-in-out 0s;
+}
+
+.stats-card::before {
+  border-bottom: 1px solid #ec5242;
+  border-left: 1px solid #ec5242;
+  transform-origin: 0 100% 0;
+}
+
+.stats-card::after {
+  border-right: 1px solid #ec5242;
+  border-top: 1px solid #ec5242;
+  transform-origin: 100% 0 0;
+}
+
+.stats-card:hover {
+  outline: 1px solid #ec5242;
+}
+
+.stats-title {
+  color: #ec5242;
+}
+
+.stats-description {
+  font-size: 14px;
+}
+
+#join-button {
+  margin: 20px;
+  color: white;
+  padding: 20px;
+  background-color: #ec5242;
+  font-weight: bold;
+  border: none;
+  border-radius: 4px;
+}
+
+/* Teacher Section */
+#teacher-section {
+  background: white;
+  padding: 20px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
+.section-intro {
+  margin-bottom: 30px;
+  border-bottom: 3px solid #ec5242;
+}
+
+#teacher-grid {
+  display: grid;
+  gap: 20px;
+}
+
+.teacher-container {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: 40% 50%;
+  gap: 20px;
+  max-width: 370px;
+}
+
+.teacher-image {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+}
+
+.teacher-name hr {
+  width: 30px;
+  background-color: #d3d3d3;
+  height: 1px;
+  border: none;
+  margin-top: 5px;
+}
+
+.teacher-subject {
+  font-style: italic;
+}
+
+.teacher-description {
+  font-size: 12px;
+  margin-top: 10px;
+}
+
+#view-more {
+  font-size: 18px;
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #d3d3d3;
+  background-color: #fafafa;
+  border-radius: 4px;
+  gap: 5px;
+}
+
+/* Sponsors Section */
+#sponsors {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  background-color: #272a31;
+  color: #8d8d8d;
+}
+
+#sponsors-container {
+  /* display:grid;
+  grid-template-columns: repeat(3, auto); */
```

Comment:
- [ ] Kindly remove the inline comments from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history.